docs/conf.py

```diff
@@ -272,34 +272,21 @@ def django_release():
 
 # -- Options for LaTeX output --------------------------------------------------
 
-# Use lualatex for Unicode support.
-latex_engine = 'lualatex'
-# Set fonts and fallbacks for CJK and Emojis.
+# Use xalatex for Unicode support.
+latex_engine = 'xelatex'
+latex_use_xindy = False
+# Set font for CJK and fallbacks for unicode characters.
 latex_elements = {
+    'fontpkg': r"""
+        \setmainfont{Symbola}
+    """,
     'preamble': r"""
-        \directlua{
-            luaotfload.add_fallback("seriffallbacks", {
-                "Noto Serif CJK SC:style=Regular;",
-                "Symbola:Style=Regular;"
-            })
-        }
-        \setmainfont{FreeSerif}[RawFeature={fallback=seriffallbacks}]
-
-        \directlua{
-            luaotfload.add_fallback("sansfallbacks", {
-                "Noto Sans CJK SC:style=Regular;",
-                "Symbola:Style=Regular;"
-            })
-        }
-        \setsansfont{FreeSans}[RawFeature={fallback=sansfallbacks}]
-
-        \directlua{
-            luaotfload.add_fallback("monofallbacks", {
-                "Noto Sans Mono CJK SC:style=Regular;",
-                "Symbola:Style=Regular;"
-            })
-        }
-        \setmonofont{FreeMono}[RawFeature={fallback=monofallbacks}]
+        \usepackage{newunicodechar}
+        \usepackage[UTF8]{ctex}
+        \newunicodechar{π}{\ensuremath{\pi}}
+        \newunicodechar{≤}{\ensuremath{\le}}
+        \newunicodechar{≥}{\ensuremath{\ge}}
+        \newunicodechar{♥}{\ensuremath{\heartsuit}}
```

Comment:
Is this necessary for every special character?
If so, that doesn't seem very nice…

Reply:
It depends what kind of characters are supported by the font, here we need to support Greek alphabet, Chinese alphabet, emojis, etc. it's tricky. I spent a lot of time trying to make it work with different font sets, packages, etc. Normally you will use for example `\pi` instead of `π` in LaTeX so here we need to make a "reverse engineering".

I'm not going to spend more time on this, so we can merge it and wait for a better solution or remove PDF builds totally :shrug: .

Reply:
LaTeX has always been tricky! 😄

Reply:
... and we still cannot be sure that it will work on RTD, so :crossed_fingers: